pages/using/wifi.md

```diff
@@ -15,20 +15,30 @@ encryption type.
 
 On the Raspberry Pi and Beaglebone, it is possible to change your wifi SSID or Passphrase after downloading the `.img`. 
 
-Currently this can be done using the small config [writer tool](https://github.com/petrosagg/resin-net-config). This is a temporary tool and will be phased out very soon.
+Currently this can be done by editing the `config.json`. This file can be found in the `resin-conf` partition on the SD card for most devices, except the Beaglebone Black and the Intel Edison. For the Beaglebone Black it can be found in the `flash-conf` partition. For the Intel Edison it can be found in in `resin-conf` once you have mounted the `config.img`.
+
+__Note:__ For both the Beaglebone black and the Intel Edison, you can only change this the wifi configuration **before you provision** the device. Trying to change these settings after provisioning will have no affect.
+
+In the `config.json` file edit the section called `files` with whatever `SSID` and `passphrase` you need.
 
-Your `network.config` file should look something like this, but `[SSID]` and `[passphrase]` should be substituted for your network settings.
 ```
-[service_home_wifi]
-Type = wifi
-Name = [SSID]
-Passphrase = [passphrase]
+"files": {
+    "network/settings": "[global]\nOfflineMode=false\n\n[WiFi]\nEnable=true\nTethering=false\n\n[Wired]\nEnable=true\nTethering=false\n\n[Bluetooth]\nEnable=true\nTethering=false",
+    "network/network.config": "[service_home_ethernet]\nType = ethernet\nNameservers = 8.8.8.8,8.8.4.4\n\n[service_home_wifi]\nType = wifi\nName = My_Wifi_Ssid\nPassphrase = my super secret wifi passphrase\nNameservers = 8.8.8.8,8.8.4.4"
+  }
 ```
 
+
 ### Multiple WiFi Connections
 
 Though we currently don't support multiple WiFi SSIDs through the user
-interface, this can be achieved by manually editing a configuration file on your SD card. See the [custom network guide][custom-network] for details on how to do this.
+interface, this can be achieved by manually editing the `config.json` file on your SD card. To add a second wifi network to the configuration simply append the following to `"network/network.config":`:
```

Comment:
maybe it's better to say "inject" or "add"? or ideally just provide the full code snippet for those not familiar with JSON? otherwise the trailing quote notice may be confusing as there's not actual quote in the snippet below.